python/lsst/cp/pipe/ptc/cpPtcExtract.py

```diff
@@ -62,7 +62,7 @@ class PhotonTransferCurveExtractConnections(pipeBase.PipelineTaskConnections,
         deferLoad=True,
     )
     taskMetadata = cT.Input(
```

Comment:
We need a TODO and a ticket to do the removal of this code that is blocked on the v28 release.